js/off.canvas.drawer.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+
+  Drupal.behaviors.megatronDrawer = {
+    attach: function(context, settings) {
+      // Primary drawer toggle.
+      var pushTrigger = document.getElementsByClassName("drawer-toggle--primary");
```

Comment:
This won't work in IE8, I'm fine with that but may want to double check with James to see what they are supporting. 
https://caniuse.com/#search=getElementsByClassName

Reply:
I don't think that should be an issue - we don't support anything older than IE10 for new projects (unless someone wants to make that a requirement, in which case we would expect to make adjustments). I figure IE9 is a very reasonable baseline these days.

Reply:
I should have prefaced that with "Of course, this is just my opinion".

Reply:
Ok cool, this review can be closed.